tests/tape/test_tape.py

```diff
@@ -728,7 +729,7 @@ def test_nesting_and_decomposition(self):
             qml.probs(wires=0), qml.probs(wires="a")
 
         new_tape = tape.expand()
-        assert len(new_tape.operations) == 5
+        assert len(new_tape.operations) == 4
```

Comment:
I thought here we do not have to expand to higher depth, but can just adjust the count...